### PR TITLE
Made a test modification for pull testing

### PR DESCRIPTION
**Pull request by sigonasr2**
Please review and update.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,7 +36,7 @@ <h1>{dev}Osan</h1>
     <!-- below the fold. Vision statement, Meeting info -->
     <div class="container-fluid">
       <div class="row">
-        <div class="col">asdf</div>
+        <div class="col">This is a test modification of the site.</div>
```
Comment (kevinfalting) [inline index.html:39]: you need to be more descriptive.

Comment (sigonasr2) [reply index.html:39]: Fine.

Comment (kevinfalting) [reply index.html:39]: Cool test bro

**Review by kevinfalting — COMMENTED** [review]
this is a general comment. great job!

**Pull request merged**